5-Cookies/client-react/src/components/TodoList/TodoListItem/TodoListItem.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, FC } from "react";
+import { ITodoItem, TodoListItem } from "../../../interfaces/interfaces";
+import { classes } from "../../../js-styles/style";
+import IconButton from "../../general/IconButton";
+import Switch from "../../general/Switch";
+import EditTodoItem from "../EditTodoItem";
+import { todosApi } from "../../../../App";
+import dataHooks from "../../../dataHooks/dataHooks";
+import "../../../../style.css";
+import { Item } from "../../../interfaces/interfaces";
+import { Guid } from "../../../../../common/interfaces/Todo";
```

Comment:
i suggest exposing everything from an index file, so consumers of the `common` types aren't aware of the internal structure

Reply:
Do you mean expose every **interface** of the app from an index.file?